apps/process_tpstream.cxx

```diff
@@ -212,6 +212,7 @@ int main(int argc, char const *argv[])
   if (latencies) {
     std::filesystem::path output_path(output_file_path);
     ta_emulator->set_timing_file((output_path.parent_path() / ("ta_timings_" + output_path.stem().string() + ".csv")).string());
+    ta_emulator->write_csv_header("TP Time Start,TP ADC Integral,Time Diffs,Is Last TP In TA");
```

Comment:
I like this, although I wish it was done in a separate PR, not in a simple "v4 to v5". I'm now having to do PR review commit-by-commit :(

Reply:
I understand the out-of-scopeness, but it was part of the requested changes. I tried to keep the commits well separated for what they changed.